xml_converter/src/icon_gen.cpp

```diff
@@ -66,6 +66,22 @@ bool Icon::init_xml_attribute(rapidxml::xml_attribute<>* attribute, vector<XMLEr
         this->color = parse_color(attribute, errors);
         this->color_is_set = true;
     }
+    else if (attributename == "alpha") {
+        this->color.alpha = parse_float(attribute, errors);
+        this->color_is_set = true;
+    }
+    else if (attributename == "blue") {
```

Comment:
https://github.com/AsherGlick/Burrito/pull/134#discussion_r1174771750

I think the discussion may go beyond this PR. Because the question being asked could just as easily apply to any attribute with multiple applicable XML values. For instance, Color currently has `xml_fields: [Color, BHColor]`.